about.html

```diff
@@ -0,0 +1,96 @@
+      <a class="header-logo" href="index.html"></a>
+      <a class="hamburger" href="#"><i id="menu-icon" class="fa-solid fa-bars" onclick="changeIcon(this)"></i></a>
+      <a class="menu-open">
+        <ul id="menu-list" class="menu-list">
+          <li><a id="list-item1" href="index.html">Home</a></li>
+          <li><a id="list-item2" href="#">Program</a></li>
+          <li><a id="list-item3" href="#">Sponsor</a></li>
+          <li><a id="list-item4" href="#">News</a></li>
+          <li><a id="list-item5" href="#">Live Score</a></li>
+          <li><a id="list-item6" href="#">Participate</a></li>
+        </ul>
+      </a>
+    </nav>
+  </header>
+
+  <section id="about-page" class="about-page">
+    <section class="about-main">
+      <div class="about-bg"></div>
+      <p class="greeting-about">Hello! Creative Minds</p>
+      <h1 class="headline-about">LOCUS: National Technical Exhibition 2023</h1>
+      <p class="short-description-about">LOCUS is the biggest Technical Fest in Nepal. It has a long history of 20 years. LOCUS is an umbrella organization led by undergraduate students of Electrical, Electronics, and Computer Engineering departments from IOE, Pulchowk Campus. All activities are guided by a single compass: fostering collaboration. Let's celebrate this March on the name of new innovation! </p>
+      <p class="contact">For more details about LOCUS 2023<br> Please feel free to contact on <br>
+        <strong><u>locus2023@pcampus.edu.np</u></strong>
+      </p>
+    </section>
+  
+    <section class="history">
+      <h2>Chronicles</h2>
+      <hr>
+      <p>Have your quick glance at the last two LOCUS events from 2022 and 2021!</p>
+      <div class="event">
+        <div class="event-one">
+          <h3 class="date">2022</h3>
+          <p>Featuring LOCUS 2022 - Year for Technology and Tnnovation</p>
+        </div>
+        <div class="event-two">
+          <h3 class="date">2021</h3>
+          <p>Featuring LOCUS 2021 - Come Out of Your Shell</p>
+        </div>
+      </div>
+    </section>
+  </section>
+
+  <section class="partner">
+    <h3>Partner</h3>
+    <hr>
+    <ul class="partner-logo">
+      <li><img src="assets/programiz.webp" alt=""></li>
+      <li><img src="assets/logpoint.webp" alt=""></li>
+      <li><img src="assets/yomari.webp" alt=""></li>
+      <li><img src="assets/naamche.webp" alt=""></li>
+      <li><img src="assets/genese.webp" alt=""></li>
+    </ul>
+  </section>
+
+  <footer>
+    <div class="wrap-footer">
+      <a class="logo" href="#"><img src="assets/logo.JPG" alt=""></a>
+      <div class="wrap-reserve">
+        <p class="reserve">LOCUS (2023), <br> All Rights Reserved.</p>
+        <p class="elaborate">No part of this publication may be reproduced, distributed, or transmitted in any form or by any means, including photocopying, recording, or other electronic or mechanical methods, without the prior written permission of the publisher.</p>
+      </div>
+    </div>
+  </footer>
```

Comment:
- [ ] [Optional] I would suggest you please work on the footer on your about section. As you have only one footer which has black background containing the logo on the left and text on right. See the image below for reference.
### Your design
![his](https://user-images.githubusercontent.com/81727988/218168675-4c164695-9886-4086-9128-3d854ba6814d.png)

### Original design
![original](https://user-images.githubusercontent.com/81727988/218168704-9599521b-65c4-4743-bf7f-036a57314c51.png)

Reply:
Thank you for pointing it out. as you suggested this change as an optional requirement, and I will work on it in the coming days.